pytorch/training/docker/1.12/py3/cu113/Dockerfile.sagemaker.gpu.os_scan_allowlist.json

```diff
@@ -1,9 +1,39 @@
 {
+  "cryptography": [
+    {
+      "description": " If an X.509 certificate contains a malformed policy constraint and policy processing is enabled, then a write lock will be taken twice recursively. On some operating systems (most widely: Windows) this results in a denial of service when the affected process hangs. Policy processing being enabled on a publicly facing server is not considered to be a common setup. Policy processing is enabled by passing the `-policy' argument to the command line utilities or by calling either `X509_VERIFY_PARAM_add0_policy()' or `X509_VERIFY_PARAM_set1_policies()' functions.",
+      "vulnerability_id": "CVE-2022-3996",
+      "name": "CVE-2022-3996",
+      "package_name": "cryptography",
+      "package_details": {
+        "file_path": "opt/conda/lib/python3.8/site-packages/cryptography-38.0.4.dist-info/METADATA",
+        "name": "cryptography",
+        "package_manager": "PYTHONPKG",
+        "version": "38.0.4",
+        "release": null
+      },
+      "remediation": {
+        "recommendation": {
+          "text": "None Provided"
+        }
+      },
+      "cvss_v3_score": 7.5,
+      "cvss_v30_score": 0,
+      "cvss_v31_score": 7.5,
+      "cvss_v2_score": 0,
+      "cvss_v3_severity": "HIGH",
+      "source_url": "https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-3996.html",
+      "source": "UBUNTU_CVE",
+      "severity": "LOW",
+      "status": "ACTIVE",
+      "title": "CVE-2022-3996 - cryptography"
+    }
+  ],
   "torch": [
     {
-      "description": "[torch](https://pypi.org/project/torch) is a Tensors and Dynamic neural networks in Python with strong GPU acceleration\n\nAffected versions of this package are vulnerable to Command Injection in `torch.jit.annotations.parse_type_line` which can cause arbitrary code execution because `eval` is used unsafely.",
-      "vulnerability_id": "SNYK-PYTHON-TORCH-3149871",
-      "name": "SNYK-PYTHON-TORCH-3149871",
+      "description": " In PyTorch before trunk/89695, torch.jit.annotations.parse_type_line can cause arbitrary code execution because eval is used unsafely.",
```

Comment:
we might want to add a "reason_to_ignore": to state why we are ignoring this Critical CVE. Example here -> https://github.com/xncqr/deep-learning-containers/blob/cf53bebe8d3fcf89259d68a90cc71912f93b71c0/pytorch/inference/docker/1.13/py3/Dockerfile.ec2.cpu.os_scan_allowlist.json#L4

One Block below would be the same.